openhands/integrations/service_types.py

```diff
@@ -18,6 +18,7 @@ class ProviderType(Enum):
     GITHUB = 'github'
     GITLAB = 'gitlab'
     BITBUCKET = 'bitbucket'
+    ENTERPRISE_SSO = 'enterprise_sso'
```

Comment:
We seem to be adding this new provider type but its never sent to the FE it seems. Is this purely for the BE, and how does the FE's `isAuthed` hook know about SSO?